src/frontend/komponenter/Fagsak/Vedtak/PdfFrame.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Document, Page } from 'react-pdf/dist/entry.webpack';
+import NavFrontendSpinner from 'nav-frontend-spinner';
+
+interface IPdfFrameProps {
+    pdfData: string;
+}
+
+const PdfFrame: React.FunctionComponent<IPdfFrameProps> = ({ pdfData }) => {
+    const [antallSider, setAntallSider] = React.useState<number>(0);
+
+    const onDocumentLoadSuccess = (pdfInfo: { numPages: React.SetStateAction<number> }) => {
+        setAntallSider(pdfInfo.numPages);
+    };
+
+    return (
+        <div className="pdf-frame">
+            <Document
+                file={pdfData}
+                onLoadSuccess={onDocumentLoadSuccess}
+                error={'Kunne ikke laste inn PDF-fil.'}
```

Comment:
Use frontendFeilmelding?

Reply:
I think the error message here is for invalid PDF data. The backend error is displayed in the upper level component.